frontend/ui/translator.lua

```diff
@@ -277,6 +277,15 @@ This is useful:
                     G_reader_settings:flipNilOrTrue("translator_from_auto_detect")
                 end,
             },
+            {
+                text = _("Show romanizations"),
```

Comment:
Could you please add also a `help_text` to explain what the feature does in plain english?  :)

I understand it could be useful for all non latin script users¿?

Reply:
I added it below this. Please review.